packages/react-core/src/components/NotificationDrawer/NotificationDrawerHeader.tsx

```diff
@@ -36,7 +44,23 @@ export const NotificationDrawerHeader: React.FunctionComponent<NotificationDrawe
     {(customText !== undefined || count !== undefined) && (
       <span className={css(styles.notificationDrawerHeaderStatus)}>{customText || `${count} ${unreadText}`}</span>
     )}
-    {children && <div className={css(styles.notificationDrawerHeaderAction)}>{children}</div>}
+    {children && (
+      <div className={css(styles.notificationDrawerHeaderAction)}>
+        {children}
+        {onClose && (
+          <div>
+            <Button
+              className="pf-c-button pf-m-plain"
```

Comment:
You should not need these classes here. The `Button` already has the `pf-c-button` class. You can just set `variant` to plain to get the `pf-m-plain` modifier applied.